test/rowtransforms.jl

```diff
@@ -0,0 +1,56 @@
+include("imports.jl")
+
+@testset ExtendedTestSet "`NormalizeRow`" begin
+    cols = [:col1, :col2, :col3]
+    item = TabularItem((; zip(cols, [1, "a", 10])...), cols)
+    cols_to_normalize = [:col1, :col3]
+    col1_mean, col1_std = 10, 100
+    col3_mean, col3_std = 100, 10
+    normdict = Dict(:col1 => (col1_mean, col1_std), :col3 => (col3_mean, col3_std))
+
+    tfm = NormalizeRow(normdict, cols_to_normalize)
+    # @test_nowarn apply(tfm, item)
+    testapply(tfm, item)
+    titem = apply(tfm, item)
+    @test titem.data[:col1] == (item.data[:col1] - col1_mean)/col1_std
+    @test titem.data[:col3] == (item.data[:col3] - col3_mean)/col3_std
+end
+
+@testset ExtendedTestSet "`FillMissing`" begin
+    cols = [:col1, :col2, :col3]
+    item = TabularItem((; zip(cols, [1, missing, missing])...), cols)
+    cols_to_fill = [:col1, :col3]
+    col1_fmval = 1000.
+    col3_fmval = 1000.
+    fmdict = Dict()
+    fmdict[:col1], fmdict[:col3] = col1_fmval, col3_fmval
```

Comment:
Why not collapse this into one line like you have above?
```suggestion
    fmdict = Dict(:col1 => col1_fmval, :col3 => col3_fmval)
```